Dikke Ploaten/Services/Database.swift

```diff
@@ -72,15 +72,15 @@ class Database {
 		}
 	}
 	
-	func deleteAlbum(albumId: String, completionHandler: @escaping () -> ()) {
+	func deleteAlbum(albumId: String, completionHandler: @escaping (Error?) -> ()) {
 		db.collection("userPlaten").document(albumId).delete() { err in
 			if let err = err {
 				print(err)
 				print("Error removing document: \(err.localizedDescription)")
+				completionHandler(err)
```

Comment:
Hier roep je de completionhandler best pas aan na de if/else, want nu krijg je enkel feedback als er een error is, zie ook comment in CollectionViewController

Reply:
Hier zou ik dit doen:

```Swift
func deleteAlbum(albumId: String, completionHandler: @escaping (Error?) -> ()) {
	db.collection("userPlaten").document(albumId).delete() { err in
		if let err = err {
			print("Error removing document: \(err.localizedDescription)")
		} else {
			print("Document successfully removed!")
		}
	completionHandler(err)
}
```